content/docs/howto/java.md

```diff
@@ -268,7 +268,7 @@ jar].
 
 After compiling and packaging, the buildpack will replace provided application
 source code with the exploded JAR and proceed as described in [Building from an
-Executable Jar][building-from-an-executable-jar].
+Executable Jar](#build-from-an-executable-jar).
```

Comment:
Would you mind setting this up as a `relref` similar to what you have starting on line 650?

Reply:
Done